troubleshoot-install.md

```diff
@@ -156,6 +156,36 @@ If this is true, you are likely to be hitting a CoreDNS routing issue. We recomm
 1. Go to <https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml#L13>
 2. Replace ```{{ $serviceName }}.{{ .Release.Namespace }}``` with ```localhost```
 
+## Issue: PodSecurityPolicy CRD is missing for `kubecost-grafana` and `kubecost-cost-analyzer-psp`
+
+PodSecurityPolicy has been [removed from Kubernetes v1.25](https://kubernetes.io/docs/concepts/security/pod-security-policy/). This will result in the following error during install.
+
+```bash
+$ helm install kubecost kubecost/cost-analyzer
+Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: [
+    resource mapping not found for name: "kubecost-grafana" namespace: "" from "": no matches for kind "PodSecurityPolicy" in version "policy/v1beta1" ensure CRDs are installed first,
+    resource mapping not found for name: "kubecost-cost-analyzer-psp" namespace: "" from "": no matches for kind "PodSecurityPolicy" in version "policy/v1beta1" ensure CRDs are installed first
+]
+```
+
+To disable PodSecurityPolicy from Kubecost's Grafana Helm Chart:
+
+```bash
+# Clone the Helm chart
+$ git clone git@github.com:kubecost/cost-analyzer-helm-chart.git
+​
+# Update Grafana's configuration for PodSecurityPolicy
+# 
+# FILE:
+# ./cost-analyzer-helm-chart/cost-analyzer/charts/grafana/values.yaml
+# 
+# CONFIG:
+# .Values.rbac.pspEnabled: false
```

Comment:
Is there a benefit to cloning the chart locally and manually updating the value in the grafana values file as opposed to just setting it directly in our chart via `.Values.grafana.rbac.pspEnabled`?

Reply:
Thank you for the catch!! I didn't realize you could set a subchart's values by doing this. Learned something new 😊. Fixed!